tests/Kernel/CorporateBlocks/CorporateBlocksFooterTest.php

```diff
@@ -5,316 +5,182 @@
-    $this->assertEquals($custom_footer_data['oe_corporate_blocks.footer_link.social.facebook']['label'], preg_replace('/[^[:print:]]/', '', trim($social_link1->text())));
+    $section = $crawler->filter('footer.ecl-footer-standardised section.ecl-footer-standardised__section8 .ecl-footer-standardised__section:nth-child(2)');
+    $actual = $section->filter('.ecl-footer-standardised__title');
+    $this->assertEquals('Social media title', $actual->last()->text());
 
-    $social_link2 = $custom_footer_social->filter('a')->eq(1);
-    $this->assertEquals($custom_footer_data['oe_corporate_blocks.footer_link.social.twitter']['url'], $social_link2->attr('href'));
-    $this->assertEquals($custom_footer_data['oe_corporate_blocks.footer_link.social.twitter']['label'], preg_replace('/[^[:print:]]/', '', trim($social_link2->text())));
+    $this->assertLinkList($section, 'standardised', $test_data['social_media']);
 
-    $social_link3 = $custom_footer_social->filter('a')->eq(2);
-    $this->assertEquals($custom_footer_data['oe_corporate_blocks.footer_link.social.other_social_media']['url'], $social_link3->attr('href'));
-    $this->assertEquals($custom_footer_data['oe_corporate_blocks.footer_link.social.other_social_media']['label'], preg_replace('/[^[:print:]]/', '', trim($social_link3->text())));
+    $section = $crawler->filter('footer.ecl-footer-standardised section.ecl-footer-standardised__section8 .ecl-footer-standardised__section:nth-child(3)');
+    $actual = $section->filter('.ecl-footer-standardised__title');
+    $this->assertEquals('Legal links title', $actual->text());
 
-    $custom_footer_other_links = $crawler->filter('footer.ecl-footer section.ecl-footer__identity div.ecl-row .ecl-col-12.ecl-col-md-6:nth-child(2)');
+    $this->assertLinkList($section, 'standardised', $test_data['legal_links']);
 
-    $other_link1 = $custom_footer_other_links->filter('a')->eq(0);
-    $this->assertEquals($custom_footer_data['oe_corporate_blocks.footer_link.general.contact']['url'], $other_link1->attr('href'));
-    $this->assertEquals($custom_footer_data['oe_corporate_blocks.footer_link.general.contact']['label'], preg_replace('/[^[:print:]]/', '', trim($other_link1->text())));
+    $section = $crawler->filter('footer.ecl-footer-standardised section.ecl-footer-standardised__section9');
 
-    $other_link2 = $custom_footer_other_links->filter('a')->eq(1);
-    $this->assertEquals($custom_footer_data['oe_corporate_blocks.footer_link.general.legal_notice']['url'], $other_link2->attr('href'));
-    $this->assertEquals($custom_footer_data['oe_corporate_blocks.footer_link.general.legal_notice']['label'], preg_replace('/[^[:print:]]/', '', trim($other_link2->text())));
-  }
+    $actual = $section->filter('.ecl-footer-standardised__title');
+    $this->assertEquals('Institution links title', $actual->text());
 
-  /**
-   * Test data for the custom footer.
-   */
-  protected function getTestCustomFooterConfigsData(): array {
```

Comment:
We are removing this as it's out of scope, maybe let's drop a comment on the issue where we will style specific footer links with a link to this line, so we make sure we look at this to implement it.

Reply:
is the ticket mentioned EWPP-468 ?